brayns/CMakeLists.txt

```diff
@@ -7,13 +7,15 @@
 add_subdirectory(parameters)
 add_subdirectory(common)
 add_subdirectory(io)
+add_subdirectory(plugin)
```

Comment:
pluginapi to match lib name?

Reply:
sure, I renamed it later and noticed we already have pluginS for the built-in ones